lib/repositories/i_movie_repository.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+import '/data_classes/movie.dart';
+
+abstract class MovieRepository {
+  Future<List<Movie>> getMovies();
+
+  Future<Movie> getMovieById(int id);
+}
+
+abstract class GenresRepository {
+  Future<String?> getFilm(int id);
```

Comment:
This should be `getGenre`, right? Same below

Reply:
Yeah, my bad. Solved